src/less/phone-input/phone-input.less

```diff
@@ -0,0 +1,141 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+
+.phone-input {
+    display: inline-flex;
+}
+
+.phone-input button.btn {
+    min-width: 64px;
+    outline: none;
+    padding-inline: var(--spacing-200) var(--spacing-100);
+}
+
+.phone-input button.btn svg.icon:last-child {
+    margin-inline-start: var(--spacing-50);
+}
+
+.phone-input .listbox-button__value {
+    & svg {
+        margin-inline-end: var(--spacing-200);
+    }
+
+    & span {
+        margin-inline-end: var(--spacing-100);
+    }
+
+    & span:last-child {
+        color: var(--color-foreground-secondary);
+    }
+}
+
+.phone-input .textbox {
+    align-items: center;
+    .background-color-token(textbox-background-color, color-background-secondary);
+    .border-color-token(textbox-border-color, color-stroke-default);
+    border-radius: 0 var(--border-radius-50) var(--border-radius-50) 0;
+    border-style: solid;
+    border-width: 1px;
+    color: var(--textbox-prefix-color, var(--color-foreground-primary));
+    display: inline-flex;
+    margin-left: -1px;
+    padding-inline-start: var(--spacing-100);
+}
+
+.phone-input .textbox > input.textbox__control {
+    border: none;
+    height: 36px;
+    padding-inline: var(--spacing-50);
+}
+
+.phone-input .textbox > input.textbox__control:focus {
+    .background-color-token(textbox-focus-background-color, color-background-secondary);
+    outline: none;
+}
+
+// stylelint-disable no-descending-specificity
+.phone-input button.btn:focus,
+.phone-input:not(.phone-input--readonly) .textbox:focus-within {
+    .border-color-token(textbox-focus-border-color, color-stroke-strong);
+    border-width: 2px;
+    height: 36px;
```

Comment:
This is adding a slight shift in the prefix number when focusing. This is because of the change in 1px border to 2px border. Unless we fix the size of the input and use `box-sizing: border-box` then it will do this strange shift. 

I think a better way to do this is to add `outline: auto` when focusing. This way the border size change doesn't shift the content inside. There is a slight problem with the border-radius but I think its way better than having the content shift when focusing. 

Another option would be to use `box-shadow`, which might be the best solution. 
<img width="1235" alt="Screenshot 2024-03-18 at 9 17 27 AM" src="https://github.com/eBay/skin/assets/1755269/fbb1f965-efee-4625-a66d-f9e7fda526e7">
What I have here doesn't cause any shifts. 

I don't think the way it currently works is good because of that internal shift.